packages/main/pyproject.toml

```diff
@@ -57,7 +57,7 @@ pynput-robocorp-fork = "^4.0.0"
 python-xlib = { version = ">=0.17", platform = "linux" }
 psutil = { version = "^5.7.0", platform = "win32" }
 pyperclip = "^1.8.0"
-exchangelib = "^4.5.1"
+exchangelib = "^4.7.5"
```

Comment:
Meanwhile, a bug affecting OAuth2 token refresh with Microsoft Exchange library was fixed. (including in this release as opportunistic refactoring)